Jellyfin.Server/Migrations/Routines/MigrateLibraryDb.cs

```diff
@@ -107,6 +107,45 @@ public void Perform()
         _logger.LogInformation("Saving BaseItems entries took {0}.", stopwatch.Elapsed);
         stopwatch.Restart();
 
+        _logger.LogInformation("Start moving ItemValues.");
+        // do not migrate inherited types as they are now properly mapped in search and lookup.
+        var itemValueQuery = "select ItemId, Type, Value, CleanValue FROM ItemValues WHERE Type <> 6";
+        dbContext.ItemValues.ExecuteDelete();
+
+        // EFCores local lookup sucks.
```

Comment:
Why? We should properly document this or in the future we won't know what we meant.